securemailbox/views/send.py

```diff
@@ -8,4 +12,45 @@
 
 @send_blueprint.route("/send/", methods=["POST"])
 def send():
+
+    #get fingerprint
+    fingerprint = request.json.get("fingerprint", None)
```

Comment:
The program throw a name error in this line. `NameError: name 'request' is not defined`
The detail is [here](https://gist.github.com/xingchengyusi/72e717235b186b65fe2176a29f73b9f9).